test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoMockTest.java

```diff
@@ -147,53 +157,15 @@ public void testOfFirstPeriod() throws Exception {
         assertScheduledActivity(activities2.get(3), ACTIVITY_1_REF, "2015-04-14T13:00:00-07:00");
         assertScheduledActivity(activities2.get(4), ACTIVITY_2_REF, "2015-04-14T13:00:00-07:00");
 
-        verify(mapper).query((Class<DynamoScheduledActivity>) any(Class.class),
-                        (DynamoDBQueryExpression<DynamoScheduledActivity>) any(DynamoDBQueryExpression.class));
-        verifyNoMoreInteractions(mapper);
-    }
-
-    @SuppressWarnings("unchecked")
-    @Test
-    public void activitySchedulerFiltersActivities() throws Exception {
-        DateTime endsOn = NOW.plus(Period.parse("P4D"));
-        Map<String, DateTime> events = Maps.newHashMap();
-        events.put("enrollment", ENROLLMENT);
-        ScheduleContext context = new ScheduleContext.Builder()
-            .withStudyIdentifier(TEST_STUDY)
-            .withClientInfo(ClientInfo.fromUserAgentCache("App/5"))
-            .withTimeZone(PACIFIC_TIME_ZONE)
-            .withEndsOn(endsOn)
-            .withHealthCode(HEALTH_CODE)
-            .withEvents(events).build();
-
-        List<ScheduledActivity> activities = TestUtils.runSchedulerForActivities(user, context);
-        mockQuery(activities);
-        List<ScheduledActivity> activities2 = activityDao.getActivities(context);
-
-        // The test schedules have these appVersions applied to them
-        // SchedulePlan DDD/Activity 1: version 2-5
-        // SchedulePlan BBB/Activity 2: version 9+
-        // SchedulePlan CCC/Activity 3: version 5-8
-        // Activity_1 and Activity_3 will match v5, Activity_2 will not. These results are 
-        // just like the next test of 4 days, but without the Activity_2 activity
-        // Activities are sorted first by date, then by label ("Activity1", "Activity2" & "Activity3")
-        assertEquals(3, activities2.size());
-        assertScheduledActivity(activities2.get(0), ACTIVITY_3_REF, "2015-04-13T13:00:00-07:00");
-        assertScheduledActivity(activities2.get(1), ACTIVITY_1_REF, "2015-04-14T13:00:00-07:00");
-        assertScheduledActivity(activities2.get(2), ACTIVITY_3_REF, "2015-04-15T13:00:00-07:00");
-
-        verify(mapper).query((Class<DynamoScheduledActivity>) any(Class.class),
-                        (DynamoDBQueryExpression<DynamoScheduledActivity>) any(DynamoDBQueryExpression.class));
+        verify(mapper).batchLoad((List<Object>)any());
         verifyNoMoreInteractions(mapper);
     }
     
     @SuppressWarnings("unchecked")
-    @Test
-    public void testOfSecondPeriodWithDifferentStartTime() throws Exception {
-        DateTime endsOn = NOW.plus(Period.parse("P4D"));
```

Comment:
It looked like activitySchedulerFiltersActivities() also tested changing the period represented by the now value, different from the testOfFirstPeriod() test (it also verifies filtering and sorting, though a number of these tests do this be examining each element of the results in order). So it seems redundant but I can add it back.